internal/log/logger.go

```diff
@@ -26,3 +33,11 @@ func (s *SimpleLogger) Debug(a ...any) {
 		fmt.Println(a...)
 	}
 }
+
+func (s *SilentLogger) Print(a ...any) {
+	// SilentLogger - Do nothing
+}
+
+func (s *SilentLogger) Debug(a ...any) {
+	// SilentLogger - Do nothing
+}
```

Comment:
Can you put these func near the `SilentLogger` and remove the comment to keep them in one line? Maybe adding the comment to the `SilentLogger` struct.

```go
// SilentLogger is a nop Logger implementation
type SilentLogger struct {}

func (s *SilentLogger) Print(a ...any) {}
func (s *SilentLogger) Debug(a ...any) {}
```

I don't have any preference about the constructor, it's not really useful without any params, but anyway.. 🙂

Reply:
Yes, off course